### ISSUE TITLE
AV1 CBR padding function

### ISSUE THREAD

**Issue by ckdt1105** [ma3uk/obs-qsv-onevpl#11]
![image](https://github.com/ma3uk/obs-qsv-onevpl/assets/74068143/f1c7e82a-038f-4a7e-be8a-9e7b2fd0f684)
With the CBR Padding feature, the bitrate on static screens doesn't drop much, so bitrate spikes are less.
(It seems to fill the remaining bitrate with meaningless information.)

In H264 and HEVC, even if the buffer size is large (bitrate X 2), the CBR Padding function works (probably), so the chance of bitrate spikes is very small.

In AV1, the bitrate is reduced on static screens, which is very likely to cause bitrate spikes.

**Comment by ckdt1105** [comment]
https://github.com/ma3uk/obs-qsv-onevpl/issues/5 is maybe mistaking it for a bug due to the lack of CBR padding in AV1.

Therefore, I think that the modification of the default buffer size to TargetBitrate / 8 in 1.1.0.0_dev2 may not be necessary.

**Comment by ma3uk** [comment]
Thanks for the Issue. I am aware of this problem, all the conditions for CBR Padding to work with AV1 were met on my part, but it did not work, so a week ago I contacted Intel and gave them all the information, at the moment I am waiting for feedback from Intel

**Comment by ckdt1105** [comment]
@ma3uk
Thanks for information.
i hope intel can fix this problem.

**Comment by ckdt1105** [comment]
@ma3uk 
i found intel finally fix cbr problem
https://github.com/obsproject/obs-studio/pull/10902

**Comment by ma3uk** [comment]
> @ma3uk i found intel finally fix cbr problem [obsproject/obs-studio#10902](https://github.com/obsproject/obs-studio/pull/10902)

Yes, I know this method, but if you look at the content of the PR that you threw off, you can see that it is designed for a new generation of GPUs starting with integrated Lunar Lake GPUs and discrete Battlemage GPUs. 
You can try to reproduce this fix by setting the WinBRCMaxAvgKbps parameters yourself to Bitrate * 1.3, WinBRCSize to a value equal to the final FPS, and also enable CustomBufferSize and set it to Bitrate * 2. 
You can test both with and without the fix enabled and post the results here, based on the results we will decide whether to enable this fix by default or leave everything as it is now, with the possibility of fine-tuning by the user.
An update has recently been made to the Intel driver repository that adds a driver-level fix starting with Meteor Lake integrated GPUs.

**Comment by ckdt1105** [comment]
@ma3uk
Thank you for your massive explain
